app/views/user_mailer/new_user_password_instructions.erb

```diff
@@ -2,4 +2,4 @@ Your profile has been created in the application. If you did not make this reque
 
 <%= @edit_new_user_password_url %>
 
-If the above URL does not work try copying and pasting it into your browser. If you continue to have problems please contact your administrator.
+If the above URL does not work try copying and pasting it into your browser. If you continue to have problem please feel free to contact us.
```

Comment:
Grammar fix:
"If the above URL does not work, try copying and pasting it into your browser. If you continue to have problems, please feel free to contact us."